news/load_autoloaded.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* load autoloading (but not yet autoloaded) xontribs
```

Comment:
this fixes the testing scenario only right?

Reply:
yes, updated